.hooks/post-merge

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+composer install
```

Comment:
shouldn't this be `php composer.phar install` to be consistent with the readme?

> Use composer provided in this repository...
> `$ php composer.phar install`
> ...or your globally installed instance.
> `$ composer install`

Reply:
I forget that we ship a `composer.phar` in this project - this is not the standard usually. Guess we can make it the phar version here for those who do not have it installed globally 👍

Reply:
Updated to use shipped `composer.phar`